### PR TITLE
Update server.js

### PR DESCRIPTION
**Pull request by jencarlucci**
Add require("http") so sample code runs without having to make any modifications.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2158)

### CODE REVIEW

File: docs/tutorials/stateless-application/server.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+var http = require("http");
```
Comment (janetkuo) [inline docs/tutorials/stateless-application/server.js]: Suggest using `'` instead of `"`(just for consistency)

**Comment by janetkuo** [comment]
Thanks for the fix! Would you sign the CLA?

**Comment by jencarlucci** [comment]
I signed it!

**Comment by janetkuo** [comment]
Oh, looks like it's just fixed on HEAD https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/tutorials/stateless-application/server.js

**Comment by jaredbhatti** [comment]
@jencarlucci  Can you double-check if you've signed the CLA?

**Comment by chenopis** [comment]
@jencarlucci I'm going to close this since it has already been fixed, but please double check that the CLA has been linked to your GitHub username. We'd love to keep getting your contributions.

* If you've already signed a CLA, it's possible we don't have your GitHub username or you're using a different email address. Check your existing CLA data and verify that your email is set on your git commits.
* If you signed the CLA as a corporation, please sign in with your organization's credentials at https://identity.linuxfoundation.org/projects/cncf to be authorized.

**Pull request closed without merging**